docs/testing.rst

```diff
@@ -263,6 +263,22 @@ This command runs a single python unit test within a specified file.
 
     pytest common/lib/xmodule/xmodule/tests/test_stringify.py::test_stringify
 
+This command runs a single python unit test method within a specified TestCase
+class within a specified file.
+
+::
+
+    pytest common/lib/xmodule/xmodule/tests/test_stringify.py::TestCase::test_stringify
```

Comment:
The only difference between this command and the one above is that this one has the class name in it.  Do both variations work?  Whether yes or no, please clarify/correct.